File: chapter_2/3_check_two_integers_part_1.py
"""
3. Check Two Integers – Part1
Design a Python program to check two given integers and return true if one of them is 30 or if their sum is 30.
"""

first_number = int(input("Enter the first number: "))
second_number = int(input("Enter the second number: "))

if (first_number == 30 or second_number == 30) or (first_number + second_number == 30):
    print(True)
else:
    print(False)

File: exercises/C/chapter_2 - Conditionals/3_check_two_integers_part_1.c
// 3. Check Two Integers – Part1
// Design a Python program to check two given integers and return true if one of them is 30 or if their sum is 30.
#include <stdio.h>

int main(int argc, char const *argv[])
{
    int first_number, second_number;

    printf("Enter first number: ");
    scanf("%d", &first_number);

    printf("Enter second number: ");
    scanf("%d", &second_number);

    if ((first_number == 30 || second_number == 30) || (first_number + second_number == 30))
        printf("True\n");
    else
        printf("False\n");

    return 0;
}
